lib/addons/keda/index.ts

```diff
@@ -74,19 +74,20 @@ export class KedaAddOn extends HelmAddOn {
   }
 
   deploy(clusterInfo: ClusterInfo): Promise<Construct> {
-    
+
     const cluster = clusterInfo.cluster;
     let values: Values = populateValues(this.options);
     values = merge(values, this.props.values ?? {});
-    
+
+    const namespace = createNamespace(this.options.namespace! , cluster);
+
     if (this.options.irsaRoles!.length > 0) {
       //Create Service Account with IRSA
       const opts = { name: this.options.kedaOperatorName, namespace: this.options.namespace };

```

Comment:
could we use `namespace` directly.

Reply:
Not sure I understand correctly
To use namespace directly, ensure that `createNamespace` is set to true. This setting is optional.

Reply:
Ignore the comment.